package.json

```diff
@@ -27,10 +29,10 @@
     "react-native-screens": "~2.2.0",
     "react-native-web": "^0.11.7",
     "react-native-webview": "8.1.1",
-    "expo-linear-gradient": "~8.1.0",
     "react-redux": "^7.2.0",
     "redux": "^4.0.5",
-    "watchman": "^1.0.0"
+    "watchman": "^1.0.0",
+    "expo-screen-orientation": "~1.0.0"
```

Comment:
Did you add this lib with expo install ?

Reply:
Nope, and this is @guillaumemoby's work ^^

Reply:
Well, I don't know

Reply:
Bu I'll remove it as we don't use it for now

Reply:
Done